eaf.el

```diff
@@ -591,35 +589,33 @@ Use it as (eaf-bind-key var key eaf-app-keybinding)"
                        (h (nth 3 window-allocation))
                        )
                   (when (and
-                         (> mouse-press-x x)
-                         (< mouse-press-x (+ x w))
-                         (> mouse-press-y y)
-                         (< mouse-press-y (+ y h)))
+                         (< x mouse-press-x (+ x w))
+                         (< y mouse-press-y (+ y h)))
```

Comment:
Why remove ```(> mouse-press-x x)``` and ```(> mouse-press-y y)``` ?

Reply:
`(and (> mouse-press-y y) (< mouse-press-y (+ y h)))` is the same as `(< x mouse-press-x (+ x w))`

Reply:
Correctly logic is:

                         (> mouse-press-x x)
                         (< mouse-press-x (+ x w))
                         (> mouse-press-y y)
                         (< mouse-press-y (+ y h))

Reply:
Oh, I see.

< support many params